mcux/drivers/lpc/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ zephyr_include_directories(.)
 
 zephyr_library_sources(fsl_common.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_LPADC		fsl_lpadc.c)
+zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_CTIMER	fsl_ctimer.c)
```

Comment:
Please fix the alignment of `fsl_ctimer` to match the other lines. Can you also make this change to `mcux/drivers/imxrt6xx/CMakeLists.txt`?

Reply:
I changed the alignment. I did mean to ask about adding that board when I spotted that it had the ctimer files. I don't have a imxrt6xx board so I will only be able to test that it compiles for that target not any of the actual behaviour.

Reply:
Don't worry about adding ctimer support on rt6xx in the main zephyr repo yet. I asked for it here so we don't have to update the west.yml manifest again.